cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -0,0 +1,2350 @@
+        initialEstimate: 1/3h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_azure_wimdows2012_ssui():
+    """
+    1. Provision service with Windows 2012 VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/3h
+        startsin: 5.8
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_ec2_rhel():
+    """
+    Add EC-2 provider.
+    Perform SSA on RHEL instance.
+    Cross-check whether smartstate instance created from AMI mentioned in
+    production.yml.
+    BZ:https://bugzilla.redhat.com/show_bug.cgi?id=1547228
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_ec2_agent_tracker():
+    """
+    BZ link:  https://bugzilla.redhat.com/show_bug.cgi?id=1557452
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_azure_windows2012r2_refs():
+    """
+    This test is to verify that you get an error when trying to perform
+    SSA on a Windows2012 r2 instance that has a ReFS formatted disk
+    attached.  Here is the before and after.
+    05/26/17 18:13:36 UTC
+    05/26/17 18:10:56 UTC
+    05/26/17 18:10:46 UTC
+    finished
+    Unable to mount filesystem. Reason:[ReFS is Not Supported]
+    Scan from Vm ReFS16on16a
+    admin
+    EVM
+    Scanning completed.
+    05/26/17 16:12:45 UTC
+    05/26/17 16:08:30 UTC
+    05/26/17 16:08:26 UTC
+    finished
+    Process completed successfully
+    Scan from Vm ReFS16on16a
+    admin
+    EVM
+    Synchronization complete
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseposneg: negative
+        initialEstimate: 1/2h
+        setup: Need to create a windows 2012r2 instance and save it onto the ReFS
+               disks on Azure.
+               Add a second disk to the VM config.  Open VM, initialize disk and
+               format it as ReFS
+               Refresh you CFME appliance
+               Perform SSA on that VM.
+               Best to just use existing ReFS vms.
+        startsin: 5.8
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_ec2_ubuntu():
+    """
+    Add EC-2 provider.
+    Perform SSA on Ubuntu instance.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_vsphere6_nested_centos_xfs_ssui():
+    """
+    1. Provision service with CentOS VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
```

Comment:
this meant to be numbered?

Reply:
Kept intentionally, will remove while automating those.